docs/resources/project_rolling_release.md

```diff
@@ -0,0 +1,95 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "vercel_project_rolling_release Resource - terraform-provider-vercel"
+subcategory: ""
+description: |-
+  Manages rolling release configuration for a Vercel project.
+---
+
+# vercel_project_rolling_release (Resource)
+
+Manages rolling release configuration for a Vercel project.
+
+## Example Usage
+
+```terraform
+resource "vercel_project" "example" {
+	name = "example-project"
+	skew_protection = "12 hours"
+}
+
+resource "vercel_project_rolling_release" "example" {
+	project_id = vercel_project.example.id
+	depends_on = [vercel_project.example]
+	rolling_release = {
+		enabled          = true
+		advancement_type = "manual-approval"
+		stages = [
+			{
+				require_approval  = true
+				target_percentage = 20
+			},
+			{
+				require_approval  = true
+				target_percentage = 50
+			},
+			{
+				require_approval  = true
+				target_percentage = 100
+			}
+		]
+	}
+}
+```
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `project_id` (String) The ID of the project.
+- `rolling_release` (Attributes) The rolling release configuration. (see [below for nested schema](#nestedatt--rolling_release))
+
+### Optional
+
+- `team_id` (String) The ID of the Vercel team.
+
+<a id="nestedatt--rolling_release"></a>
+### Nested Schema for `rolling_release`
+
+Required:
+
+- `enabled` (Boolean) Whether rolling releases are enabled.
+
+Optional:
+
+- `advancement_type` (String) The type of advancement between stages. Must be either 'automatic' or 'manual-approval'. Required when enabled is true.
+- `stages` (Attributes List) The stages of the rolling release. Required when enabled is true. (see [below for nested schema](#nestedatt--rolling_release--stages))
+
+<a id="nestedatt--rolling_release--stages"></a>
+### Nested Schema for `rolling_release.stages`
+
+Required:
+
+- `target_percentage` (Number) The percentage of traffic to route to this stage.
+
+Optional:
+
+- `duration` (Number) The duration in minutes to wait before advancing to the next stage. Required for all stages except the final stage when using automatic advancement.
+- `require_approval` (Boolean) Whether approval is required before advancing to the next stage.
+
+## Import
+
+Import is supported using the following syntax:
+
+```shell
+# If importing into a personal account, or with a team configured on
+# the provider, simply use the project ID.
+# - project_id can be found in the project `settings` tab in the Vercel UI.
+terraform import vercel_project.example prj_xxxxxxxxxxxxxxxxxxxxxxxxxxxx
+
+# Alternatively, you can import via the team_id and project_id.
+# - team_id can be found in the team `settings` tab in the Vercel UI.
+# - project_id can be found in the project `settings` tab in the Vercel UI.
+terraform import vercel_project.example team_xxxxxxxxxxxxxxxxxxxxxxxx/prj_xxxxxxxxxxxxxxxxxxxxxxxxxxxx
+```
```

Comment:
this is wrong as it's showing how to import a project, not a vercel_project_rolling_releases_thing